data/json/items/ammo/762x25.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "id": "762_25",
+    "type": "AMMO",
+    "name": "7.62x25mm JHP",
+    "name_plural": "7.62x25mm JHP",
+    "description": "A commercial version of the 7.62x25mm cartridge created for the armed forces of Soviet Russia.  It was derived from the 7.63x25mm cartridge used by the C96 pistol.",
+    "weight": 6,
+    "volume": 1,
+    "price": 10000,
+    "count": 50,
+    "stack_size": 200,
+    "material": [ "steel", "powder" ],
+    "symbol": "=",
+    "color": "light_gray",
+    "ammo_type": "762x25",
+    "casing": "762_casing",
+    "damage": 24,  "//": "The stats are copied from GAME_BALANCE.md. Muzzle energy up to 760 Joules is possible.",
+    "range": 13,  "//": "Placeholder.",
+    "pierce": 2,
+    "dispersion": 120,
+    "recoil": 195,
+    "effects": [ "COOKOFF" ]
+  },
+  {
+    "id": "762_25typeP",
+    "type": "AMMO",
```

Comment:
Please use the `copy-from` syntax to reduce duplication. Examples can be found in the `ammo` subdir